pkg/gds/admin_test.go

```diff
@@ -920,6 +921,7 @@ func (s *gdsTestSuite) TestReview() {
 func (s *gdsTestSuite) TestResend() {
 	s.LoadFullFixtures()
 	defer s.ResetFullFixtures()
+	defer emails.PurgeMockEmails()
```

Comment:
Guessing this just deletes temp files?

Reply:
We are storing the "mocked" emails as raw bytes in a [][]byte slice. Since it's global, any test that writes to it needs to clear it so we can maintain independence between tests. I had forgotten to do that when I originally wrote this test.